spring-cloud-dataflow-admin-local/pom.xml

```diff
@@ -34,8 +34,8 @@
 				<directory>src/main/resources</directory>
 				<filtering>true</filtering>
 				<includes>
-					<include>META-INF/spring.factories</include>
 					<include>admin.yml</include>
+					<include>META-INF/spring.factories</include>
```

Comment:
yes, makes sense as there's only one resource set... if we were expecting any issues from filtering (we shouldn't in this case), then we would need to provide a non-filtering resource set as well